tests/10-EdvClient.spec.js

```diff
@@ -64,6 +64,72 @@ describe('EdvClient', () => {
     docs.length.should.equal(1);
   });
 
+  it('find a document using a multi property query', async () => {
+    const client = await mock.createEdv();
+    client.ensureIndex({attribute: 'content.someKey'});
+
+    const testId = await EdvClient.generateId();
+    const doc = {
+      id: testId,
+      content: {
+        someKey: {
+          b: 5,
+          a: 4
+        },
+        indexedKey: 'value1'
```

Comment:
In the spirit of simplification, this `indexedKey` property in the document can be removed entirely.

Also, one thing we have not yet discussed is that once the PR process has begun, it's important not to rebase.  If one rebases during a PR review process, github features that involve displaying changes related to comments and changes since one last viewed the PR do not work anymore.  We'll discuss more later today.